package.json

```diff
@@ -32,6 +32,7 @@
     "eslint-plugin-cypress": "^3.0.3",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-security": "~3.0.0",
+    "json-loader": "^0.5.7",
```

Comment:
Not seeing this used anywhere so I think we can remove

Reply:
It was to handle the json data, but it ended up not getting used